deploy/ray-operator/Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Build the manager binary
+FROM golang:1.12.5 as builder
+
+WORKDIR /workspace
+# Copy the Go Modules manifests
+COPY go.mod go.mod
+COPY go.sum go.sum
+# cache deps before building and copying source so that we don't need to re-download as much
+# and so that source changes don't invalidate our downloaded layer
+RUN go mod download
+
+# Copy the go source
+COPY main.go main.go
+COPY api/ api/
+COPY controllers/ controllers/
+
+# Build
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o manager main.go
```

Comment:
Can we use another name? manager is confusing. Maybe ray-operator

Reply:
Word "manager" in Kubebuilder is a general word to describe the controller. Can you make it clear why it is confusing?

Reply:
Kubernetes has is own controller-manager. If there is only one controller managed by the third-party manager, prefer xxx-operator to avoid conflicts.

Reply:
Sounds great, thanks